packages/block-editor/src/components/block-draggable/index.native.js

```diff
@@ -156,6 +160,19 @@ const BlockDraggableWrapper = ( { children } ) => {
 		}
 	};
 
+	const onStopDragging = () => {
+		if ( currentClientId.current ) {
+			onBlockDrop( {
+				// Dropping is only allowed at root level
+				srcRootClientId: '',
```

Comment:
The root block list doesn't have a client Id so an empty string is used. In the future, once we support inner blocks, we should provide the proper root client Id.